src/foam/dao/NodeModelFileDAO.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+foam.CLASS({
+  package: 'foam.dao',
```

Comment:
I'm not sure this it the right package for this.  Especially since its such an impoverished DAO.  Perhaps a foam.classloader package would be more appropriate.

I also think this should be added to files.js with a nodejs flag.